### ISSUE TITLE
can atlas be used to reconstruct models with photos not videos?

### ISSUE THREAD

**Issue by Yajiang** [magicleap/Atlas#11]
thanks for your work at first. We tried with atlas with our data captured using our devices but it generate nothing at last. we don't feed atlas with videos or continuous images, instead we feed it with photo sequences with poses. any reply will be appreciated.

**Comment by zmurez** [comment]
Yes. Atlas is indifferent to whether the images come from a video sequence or an unstructured set.
My guess for why you are not getting results is that the poses are not in the format expected by Atlas.
The poses must be metric (in meters) with Z-up. Also the poses must be shifted so that the scene lies within the reconstruction voxel volume. (the voxel origin is [0,0,-.5] by default )

see:
https://github.com/magicleap/Atlas/blob/bee3ddbdb0935cf9aa9fc810e829115c6ab74dc5/inference.py#L56
